server/utils/applications/utils.ts

```diff
@@ -271,6 +273,20 @@ const mapTimelineEventsForUi = (timelineEvents: Array<TimelineEvent>): Array<UiT
     })
 }
 
+const mapTimelineUrlsForUi = (timelineUrls: Array<TimelineEventAssociatedUrl>) => {
+  return timelineUrls.map(item => ({ url: item.url, type: urlTypeForUi(item.type) }))
+}
+
+const urlTypeForUi = (type: TimelineEventUrlType) => {
+  const translations: Record<TimelineEventUrlType, string> = {
+    application: 'application',
+    assessment: 'assessment',
+    booking: 'booking',
+    assessmentAppeal: 'appeal',
+  }
+  return translations[type]
+}
```

Comment:
Given this is only tested through `mapTimelineUrlsForUi`, does it need to be a function rather than a POJO (whether that POJO is inside or outside of the other function)?